smithy-rules-engine/src/main/java/software/amazon/smithy/rulesengine/language/syntax/expressions/functions/ParseUrl.java

```diff
@@ -174,10 +174,6 @@ public static boolean isIpAddr(String host) {
                         || (ch2 < '0' || ch2 > '9')) {
                     return false;
                 }
-                int value = ((ch0 - '0') * 100) + ((ch1 - '0') * 10) + (ch2 - '0');
```

Comment:
What about adding a comment here about the heuristic? Otherwise someone is going to think that we can do better and attempt to change it.

Reply:
Sure